Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -1,40 +1,77 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using System.Text;
+
+namespace BootCamp.Chapter
 {
     /// <summary>
     /// Part 1.
     /// </summary>
     public static class TextTable
     {
         /*
-
          Input: "Hello", 0
            +-----+
            |Hello|
            +-----+
-           
+        */
+
+        public static string Build(string message, int padding)
+        {
+            var sb = new StringBuilder();
+            sb.Append($"+-----+{Environment.NewLine}|{ message}|{Environment.NewLine}+-----+");
+            Console.WriteLine(sb.ToString());
+            return sb.ToString();
+        }
+
+        /*
          Input: $"Hello{Environment.NewLine}World!", 0
            +------+
            |Hello |
            |World!|
            +------+
-           
+         */
+
+        public static string Build1(string message, int padding)
+        {
+            var sb = new StringBuilder();
+            sb.AppendLine($"+------+");
+            sb.AppendLine($"|{message.Substring(0, 5).PadRight(6)}|");
+            sb.AppendLine($"|{message.Substring(6, 7).Trim()}|");
+            sb.AppendLine($"+------+");
+            Console.WriteLine(sb.ToString());
+            return sb.ToString();
+        }
+
+
+        /*
          Input: "Hello", 1
            +-------+
            |       |
            | Hello |
            |       |
            +-------+
-
          */
 
+        public static string Build2(string message, int padding)
```

Comment:
You're supposed to do table build in one method.

Reply:
I need help. I don't know how to build a table around a string and use the padding int that is passed in.